en/hardware/porting_guide.md

```diff
@@ -46,6 +46,25 @@ make px4fmu-v5_default menuconfig
 make px4fmu-v5_default qconfig
 ```
 
+If you are starting just now and only used the Ubuntu installation script ([ubuntu_sim_nuttx.sh](https://raw.githubusercontent.com/PX4/Devguide/master/build_scripts/ubuntu_sim_nuttx.sh) from https://dev.px4.io/en/setup/dev_env_linux.html) - you might need to build and install one of the kconfig tools from [NuttX tools](https://bitbucket.org/nuttx/tools/src/master/).
```

Comment:
Drop might? We know the Ubuntu install script doesn't install kconfig tools, but it would be available with docker or MacOS brew instructions.

Reply:
I included the wording "might" because someone who is already working with NuttX on their machine potentially already has that installed. In my case, I was able to compile and use qconfig without explicitly installing Qt dependencies for this specifically. I would seem I had them from some other project (possibly).

Reply:
I think this should be a note eg:


---

**Note:** For fresh ubuntu installs from [ubuntu_sim_nuttx.sh](https://raw.githubusercontent.com/PX4/Devguide/master/build_scripts/ubuntu_sim_nuttx.sh) you need to install `kconfig` tools from [NuttX tools](https://bitbucket.org/nuttx/tools/src/master/).